charts/gardener/controlplane/values.yaml

```diff
@@ -93,6 +93,10 @@ global:
         memory:
           value: "5G"
           percentage: 80
+    # Comma-delimited list of admission plugins that should be disabled although they are in the default enabled plugins list.
+    disableAdmissionPlugins: ""
```

Comment:
Would it be more convenient to make this a list of strings and then join them with `,` in the template?

Reply:
+1
Maybe we can keep it similar to the way we do it in the Shoot's kube-apiserver chart.
https://github.com/gardener/gardener/blob/ca6d53cd358f2c415101aa63b1212841cddb140f/charts/seed-controlplane/charts/kube-apiserver/templates/kube-apiserver.yaml#L79